### PR TITLE
🗞️ Update dotenv to access .env mnemonic, example rpcs

### PR DESCRIPTION
**Pull request by TRileySchwarz**
-Update a couple of the hardhat config network rpcs, as existing ones were not working

-Provide instructions in example readmes to provide and fund their deployer address

### CODE REVIEW

**Comment by janjakubnanista** [comment]
🗞️ is for a feature!

**Review by janjakubnanista — CHANGES_REQUESTED** [review]
Run `pnpm changeset` and mark `create-lz-oapp` as having a patch change pls

File: examples/oapp/README.md [hunk 1]
```diff
@@ -73,3 +73,31 @@ Or adjust the `package.json` to for example remove `hardhat` tests:
...
 + "test": "forge test"
 ```
+
+## 2) Deploying Contracts
+
+Set up deployer wallet/account:
+
+> - Rename `.env.example` -> `.env`
+> - Replace the existing 12 word mnemonic with yours, this will be used to deploy your contracts.
+> - Fund this address with the corresponding chain's native tokens you want to deploy to.
```
Comment (janjakubnanista) [inline examples/oapp/README.md:83]: This in markdown means a blockquote, I think you might just want a list (remove the `>`)

Also we now have either `MNEMONIC` or `PRIVATE_KEY`, maybe we could keep this one vague (e.g. follow the instructions in `.env.example`) or provide two examples:

#### For mnemonic-based authentication

```
MNEMONIC="bla bla bla"
```

#### For private-key-based authentication

```
PRIVATE_KEY=0x0x0xxxx
```

**Review by janjakubnanista — CHANGES_REQUESTED** [review]

File: examples/oapp/README.md (hunk 1 above)
Comment (TRileySchwarz) [reply examples/oapp/README.md]: updated, please check

**Review by janjakubnanista — APPROVED** [review]

**Pull request merged**